app/template/rest.py

```diff
@@ -35,10 +36,24 @@ def _content_count_greater_than_limit(content, template_type):
     return template.content_count > current_app.config.get('SMS_CHAR_COUNT_LIMIT')
 
 
+def _has_service_permission(template_type, action, permissions):
```

Comment:
error response is slightly different as notifications have a 400 response and templates have a 403 response.